crates/sui-single-node-benchmark/src/single_node.rs

```diff
@@ -156,8 +158,21 @@ impl SingleValidator {
                     .await;
                 response.signed_effects.into_data()
             }
+            Component::TxnSigning => unreachable!(),
         };
         assert!(effects.status().is_ok());
         effects
     }
+
+    pub async fn sign_transaction(&self, transaction: Transaction) -> HandleTransactionResponse {
+        let result = self
+            .get_validator()
+            .handle_transaction(
+                &self.epoch_store,
+                VerifiedTransaction::new_unchecked(transaction),
```

Comment:
Not 100% sure this is the right way to make a transaction verified in the benchmark.